integr8ly/integreatly-operator
# allow admin accesss to user sso for multi tenant installs

# Issue link
https://issues.redhat.com/browse/MGDAPI-2649
https://issues.redhat.com/browse/MGDAPI-2653

# What
allow admin accesss to user sso for multi tenant installs

# Verification steps MGDAPI-2649
Install multi tenant RHOAM
` INSTALLATION_TYPE=multitenant-managed-api make cluster/prepare/local`
` INSTALLATION_TYPE=multitenant-managed-api make code/run`
Install testingIDP
`NUM_REGULAR_USER=10 NUM_ADMIN=5 REALM_DISPLAY_NAME="DevSandbox" REALM="devsandbox" DEDICATED_ADMIN_PASSWORD=Password1 PASSWORD=Password1 ./scripts/setup-sso-idp.sh`
Log into cluster as test-user01 - this initiates the tenant creation.
Log into the cluster as kubeadmin
Open UserSSO and confirm login possible with admin/pw from credential-rhssouser
Create test-user01 dev namespace
```
oc new-project test-user01-dev
oc adm policy add-role-to-user admin test-user01 -n test-user01-dev
```
After test-user01 tenant created, log into user-sso as test-user01, follow launcher link in namespace
Verify that the user is not prompted for username/password. i.e. only sso login 
Verify they can successfully log into their realm

# Verification steps MGDAPI-2653
Verify 3Scale console link in the application menu is not present for tenants.
